rootless/specconv/specconv_linux.go

```diff
@@ -19,10 +21,13 @@ func ToRootless(spec *specs.Spec, v2Controllers []string) error {
 func getCurrentOOMScoreAdj() int {
 	b, err := ioutil.ReadFile("/proc/self/oom_score_adj")
 	if err != nil {
+		logrus.WithError(err).Warn("failed to read /proc/self/oom_score_adj")
 		return 0
 	}
-	i, err := strconv.Atoi(string(b))
+	s := string(b)
+	i, err := strconv.Atoi(strings.TrimSpace(s))
 	if err != nil {
+		logrus.WithError(err).Warn("failed to parse /proc/self/oom_score_adj (%q)", s)
```

Comment:
Should we log the value _before_ trimming? (also simplifies the code a bit)

```suggestion
	i, err := strconv.Atoi(strings.TrimSpace(string(b)))
	if err != nil {
		logrus.WithError(err).Warn("failed to parse /proc/self/oom_score_adj (%q)", b)
```

Reply:
Already logging untrimmed string

Reply:
oh! 🤦 yes, it's just the extra variable to convert `[]byte` to string